desktop/core/src/desktop/management/commands/rungunicornserver.py

```diff
@@ -112,10 +115,16 @@ def load(self):
     return self.load_wsgiapp()
 
 def argprocessing(args=[], options={}):
+  global PID_FILE
   if options['bind']:
+    http_port = "8888"
     bind_addr = options['bind']
+    if ":" in bind_addr:
+      http_port = bind_addr.split(":")[1]
+    PID_FILE = "/tmp/hue_%s.pid" % (http_port)
```

Comment:
Should we keep the pid file in the log location as well?

Reply:
generally, PID files are kept in /tmp because when the machine gets rebooted the PID files get cleared automatically.